lib/bindConnectors/index.js

```diff
@@ -56,13 +57,15 @@ module.exports = function (config, options) {
 				messageHooks[message.name+'_response'] = bindTriggerResponse(message, options);
 			}
 
+			if (_.isFunction(message.dynamicOutput)) {
+				messageHooks[message.name + '_dynamic_output'] = bindDynamicOutput(message, options);
```

Comment:
I think there has been a misunderstanding because the message we're fetching from the front end is `_output_schema`, not `dynamic_output`.